projects/map/src/lib/map-service/map.service.ts

```diff
@@ -110,9 +110,10 @@ export class MapService {
     featureGeometry$: Observable<FeatureModelType<T> | Array<FeatureModelType<T>>>,
     vectorLayerStyle?: MapStyleModel | ((feature: FeatureModel<T>) => MapStyleModel),
     zoomToFeature?: boolean,
+    updateWhileAnimating: boolean = true,
```

Comment:
Maybe better to just make it optional and only put it to true if you actually want it, which is only for GPS at this moment I think?